samples/sensor/sgp40_sht4x/overlay-usb.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_USB=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr USB console sample"
```

Comment:
Copy/paste error?

Reply:
I have removed the USB related stuff from the sample code, but yes :)